lib/casclient.rb

```diff
@@ -3,10 +3,12 @@
 require 'logger'
 require 'net/https'
 require 'rexml/document'
-require 'casclient/dice_bag/cas_template'
-require 'active_support'
-require 'active_model_memcache_store'
-require 'active_model_redis_store'
+
+# require 'casclient/dice_bag/cas_template'
+
+# require 'active_support'
+# require 'active_model_memcache_store'
+# require 'active_model_redis_store'
```

Comment:
Can you change these to require conditionally instead of just commenting out?

```suggestion
require 'casclient/dice_bag/cas_template' if defined?(::DiceBag)
require 'active_model_memcache_store' if defined?(:: Dalli)
require 'active_model_redis_store' if defined?(::Redis)
```

Reply:
Also, please add `require 'active_support'` to the files it is in use:
- https://github.com/mdsol/rubycas-client/blob/a22876177ecc8f2a8eb26ae8b5c7872e14b69ecc/lib/casclient/frameworks/rails/filter.rb#L1
- https://github.com/mdsol/rubycas-client/blob/a22876177ecc8f2a8eb26ae8b5c7872e14b69ecc/lib/active_model_redis_store.rb#L6
- https://github.com/mdsol/rubycas-client/blob/a22876177ecc8f2a8eb26ae8b5c7872e14b69ecc/lib/active_model_memcache_store.rb#L3